src/translations/translations.fr.js

```diff
@@ -705,6 +705,14 @@ export const translations = {
   'cc-env-var-linked-services.loading.app': /** @param {{appName: string}} _ */ ({ appName }) =>
     sanitize`Chargement de la configuration publiée par les applications liées à <strong>${appName}</strong>…`,
   //#endregion
+  //#region cc-feature-list
+  'cc-feature-list.documentation-link': `Voir la documentation`,
+  'cc-feature-list.error': `Une erreur s'est produite lors du chargement des fonctionnalités expérimentales disponibles.`,
```

Comment:
nitpick: we can remove the trailing dot:

```suggestion
  'cc-feature-list.error': `Une erreur s'est produite lors du chargement des fonctionnalités expérimentales disponibles`,
```